src/pages/ItemList/components/Product.js

```diff
@@ -33,7 +33,7 @@ export const Product = ({ id, name, thumbnail, price, contents }) => {
     <div className="product" onClick={handleClickItem}>
       <div className="productCard">
         <img src={thumbnail} alt="product" className="productImage" />
-        <div className="cartIcon" onClick={handleClickCart}>
+        <div className="cartIcon" onClick={e => handleClickCart(e, id, name)}>
```

Comment:
```suggestion
        <div className="cartIcon" onClick={handleClickCart}>
```
이미 handleClickCart 라는 함수안에서 id, name 값을 컴포넌트에서 받은 props 값을 사용하고 있기에
인자로 넘겨주지 않아도 됩니다!